testdata/README.md

```diff
@@ -0,0 +1,21 @@
+# Test data
```

Comment:
How do we generate this test data? We need some rough instructions or information about this test data so that others can reproduce this library if one person is away.

Reply:
That's the next step, and possibly an internal one. I would really like to make progress rather than try to boil the ocean.